### PR TITLE
Read the Docs presence

### PR DESCRIPTION
**Pull request by marzeq**

### CODE REVIEW

File: websites/R/ReadTheDocs/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    {"title": "Main page", "value": "Viewing the main page", "id": "main"},
+    {"title": "On some documentation", "value": "Viewing %name%'s docs", "id": "viewing_docs"},
+    {"title": "On main documentation", "value": "Viewing the main docs", "id": "viewing_main_docs"},
+    {"title": "Search for", "value": "Searching for %term%", "id": "searching_for"},
+    {"title": "Search", "value": "Searching the docs", "id": "searching"},
+    {"title": "Signup", "value": "Creating an account", "id": "signup"},
+    {"title": "Login", "value": "Logging in", "id": "login"},
+    {"title": "Account settings", "value": "Managing their account", "id": "manage"},
+    {"title": "Profile", "value": "Viewing %name%'s profile", "id": "profile"},
+    {"title": "Dashboard", "value": "Viewing their dashboard", "id": "dashboard"}
```
Comment (Bas950) [inline websites/R/ReadTheDocs/dist/metadata.json:34]: format it like normal
and string inputs need placeholders.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/R/ReadTheDocs/dist/metadata.json (hunk 1 above)
Comment (marzeq) [reply websites/R/ReadTheDocs/dist/metadata.json]: Fixed

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Comment by marzeq** [comment]
> Upload proof of it working

[Here](https://youtu.be/b3rf5CFuvXw)
There are a couple of cuts, but I was just checking if there's any confidential stuff on some of the pages

**Review by Bas950 — APPROVED** [review]
Upload proof of it working

**Comment by marzeq** [comment]
> > Upload proof of it working
> 
> [Here](https://youtu.be/b3rf5CFuvXw)
> There are a couple of cuts, but I was just checking if there's any confidential stuff on some of the pages

I already did in this video https://youtu.be/b3rf5CFuvXw, if you want to, I can provide screenshots too

**Review by ririxidev — APPROVED** [review]
Codacy is good just didn't change status on github for no reason lmao

**Pull request merged**